doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -185,6 +187,7 @@ OSSL_CMP_CTX_reinit() prepares the given I<ctx> for a further transaction by
 clearing the internal CMP transaction (aka session) status, PKIStatusInfo,
 and any previous results (newCert, newChain, caPubs, and extraCertsIn)
 from the last executed transaction.
+It also clears any ITAVs added by calling OSSL_CMP_CTX_push0_genm_ITAV().
```

Comment:
This sentence completely confused me, since it can be read that the reinit gets rid of ITAVS By doing a call to OSSL_CMP_CTX_push0_genm_ITAV().

Maybe..
'It also clears any ITAVs that were added by OSSL_CMP_CTX_push0_genm_ITAV().'

Reply:
Re-phrased as suggested, also the other occurrence below.
@t8m, could you please re-approve.